### ISSUE TITLE
Lists missing label on select element when aria-label is present. #9294

### ISSUE THREAD

**Issue by leegee** [dequelabs/axe-core#1663]
Expectation: Respect `aria-label` attributes and not report those present as absent

Actual: See screenshot — `aria-label` ignored, at least on `select` elements.

Motivation: Fit the spec

<pre><code>
axe-core version: ??? How can I find this from Lighthosue?
Lighthouse version: 4.3.1

Browser and Assistive Technology versions

For Tooling issues:
- Node version: Not relevant
- Platform:   OSX 10.14.5
</code></pre>

I see this issue using the Lighthosue Chrome extension: after [reporting it to their repo](https://github.com/GoogleChrome/lighthouse/issues/9294), they asked me to file it here.

![image](https://user-images.githubusercontent.com/512816/60341451-7b136400-99a6-11e9-947b-e71f36852114.png)

**Comment by straker** [comment]
Thanks for the issue. Unfortunately I'm unable to reproduce this using just the provided `<select>` element. Do you have a URL I could test that reproduces the error?

```html
<select name="results_sort" class="ui-form__select" id="lsrp-sort-order-form-sort-order-options-fieldset-results_sort-field-0" aria-label="Sort Order"></select>
```

Ran in both axe v3.2.2 and latest axe-coconut.

**Comment by leegee** [comment]
I'm afraid it's a corporate client. If they can give me time, I will try to
get a page subset

Thanks
